pronto/api/signatures/matrices.py

```diff
@@ -13,19 +13,43 @@ def get_matrices(accessions):
     con = utils.connect_pg(utils.get_pg_url())
     cur = con.cursor()
 
-    signatures, comparisons = get_comparisons(cur, accessions)
+    signatures, comparisons, exclusive = get_comparisons(cur, accessions)
 
     cur.close()
     con.close()
     
     return jsonify({
         "signatures": signatures,
-        "comparisons": comparisons
+        "comparisons": comparisons,
+        "exclusive": exclusive
     })
 
 
 def get_comparisons(cur, accessions: tuple):
-    
+
+    exclusive = {}
+    signatures_proteins = cur.execute(
+        f"""
+        SELECT signature_acc, array_agg(protein_acc) AS proteins
+        FROM signature2protein
+        WHERE signature_acc in ({','.join(['%s'] * len(accessions))})
+        GROUP BY signature_acc 
+        """, accessions 
+    ).fetchall()
```

Comment:
We don't need to store the result of the query in a variable; we can iterate over `fetchall()` later.

```suggestion
    cur.execute(
        f"""
        SELECT signature_acc, array_agg(protein_acc) AS proteins
        FROM signature2protein
        WHERE signature_acc in ({in_params})
        GROUP BY signature_acc 
        """,
        accessions
    )
```